scripts/delete_files_with_no_metadata/delete_files_with_no_metadata.py

```diff
@@ -0,0 +1,107 @@
+import json
+import time
+
+import requests
+import subprocess
+
+DRY_RUN = False
+TOKEN = 'insert-token-without-bearer-prefix'
+SUBMISSION_URL = 'https://api.ingest.archive.data.humancellatlas.org/submissionEnvelopes/insert-submission-id'
+SUBMISSION_UUID = 'insert-submission-uuid'
+FILES_TO_DELETE_FILENAME = f'{SUBMISSION_UUID}_files_to_delete.json'
+
+
+def run(command: str, input: str = None, verbose: bool = True):
+    parsed_command = command.split(' ')
+    proc = subprocess.Popen(parsed_command,
+                            stdin=subprocess.PIPE,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            )
+    if input:
+        stdout, stderr = proc.communicate(input.encode())
+    else:
+        stdout, stderr = proc.communicate()
+
+    if verbose:
+        print('$ ' + command)
+        print(stdout.decode())
+        print(stderr.decode())
+
+    return proc.returncode, stdout.decode(), stderr.decode()
+
+
+def remove_file(s3_file_path):
+    if DRY_RUN:
+        command = f'echo "Removing file {s3_file_path}"'
+    else:
+        command = f'aws s3 rm {s3_file_path} --profile embl-ebi'
+
+    exit_code, output, error = run(command)
+
+    if exit_code == 0:
+        return f'Removed {s3_file_path}'
+    else:
+        return error
+
+
+def get_draft_files_in_submission(submission_url):
+    api_base = submission_url.split("/")[2]
+    draft_url = f"https://{api_base}/files/search/findBySubmissionEnvelopeAndValidationState?envelopeUri={submission_url}&state=DRAFT&size=200"
```

Comment:
What if there are more than 200 files? I suggest to pass a bigger number than that.

Reply:
Good catch! Will fix this.

Reply:
Fixed.